RELEASE/scripts/sl_ascend.ash

```diff
@@ -285,7 +287,7 @@ boolean handleFamiliar(string type)
 		}
 	}
 
-	if(sl_beta())
+	if(get_property("sl_beta_test").to_boolean())
```

Comment:
I know you didn't intentionally do this, but please fix it.

Reply:
Will do, I figured out what went wrong.